### PR TITLE
[CFE-403] Fix(components): voting power chart

### PR DESCRIPTION
**Pull request by Poafs1**
Please refer to this ticket: https://linear.app/alleslabs/issue/CFE-403/[bug]-validator-details

### CODE REVIEW

File: src/lib/services/validator.ts [hunk 1]
```diff
@@ -103,7 +103,9 @@ export type HistoricalPowersItem = z.infer<typeof zHistoricalPowersItem>;
 
 export const zHistoricalPowersResponse = z
   .object({
-    items: z.array(zHistoricalPowersItem),
+    items: z
+      .array(zHistoricalPowersItem)
+      .transform((items) => [...items].reverse()),
```
Comment (songwongtp) [inline src/lib/services/validator.ts:108]: revert, let's change this in the api

**Review by songwongtp — COMMENTED** [review]
missing changelog

File: src/lib/services/validator.ts (hunk 1 above)
Comment (Poafs1) [reply src/lib/services/validator.ts]: 👍🏼

File: src/lib/pages/validator-details/components/bonded-token-changes/VotingPowerChart.tsx [hunk 2]
```diff
@@ -94,20 +65,24 @@ export const VotingPowerChart = ({
...
-    ? `${getTokenLabel(singleStakingDenom, assetInfo?.symbol)}`
-    : "";
-
   const customizeTooltip = (tooltip: TooltipModel<"line">) => {
     const { raw, dataIndex } = tooltip.dataPoints[0];
 
+    const currency = singleStakingDenom
+      ? `${getTokenLabel(singleStakingDenom, assetInfo?.symbol)}`
+      : "";
+
```
Comment (evilpeach) [inline src/lib/pages/validator-details/components/bonded-token-changes/VotingPowerChart.tsx:74]: Move to outside

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**